roles/ovirt-ova-pack/files/pack_ova.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+import os
+import io
+import tarfile
+import sys
+import time
+
+BLOCKSIZE = 512
+NUL = "\0"
+buf = bytearray(4096)
+
+def createTarInfo(name, size):
+    info = tarfile.TarInfo(name)
+    info.size = size
+    info.mtime = time.time()
+    return info
+
+def padToBlockSize(file, size):
+    remainder = size % BLOCKSIZE
+    if remainder > 0:
+        file.write(NUL * (BLOCKSIZE - remainder))
+
+ova_path = sys.argv[1]
+print "opening for write: %s" % ova_path
+ova_fd = os.open(ova_path, os.O_WRONLY)
+ova_file = io.FileIO(ova_fd, "w")
+
+ovf = sys.argv[2]
```

Comment:
At the script call in playbook, you don't require this parameter to be defined, so here it can fail.

Reply:
how can enforce its definition? note that while this playbook is only used by the engine, these kind of validations are less important since they are done on the engine side